devops/actions/llvm_test_suite/action.yml

```diff
@@ -55,24 +48,14 @@ runs:
       mkdir toolchain
       tar -xf ${{ inputs.sycl_archive }} -C toolchain
       rm -f ${{ inputs.sycl_archive }}
-  - name: Download LIT
-    uses: actions/download-artifact@v3
-    with:
-      name: ${{ inputs.lit_artifact }}
-  - name: Extract LIT
-    shell: bash
-    run: |
-      mkdir lit
-      ls -la
-      tar -xf ${{ inputs.lit_archive }} -C lit
-      rm -f ${{ inputs.lit_archive }}
   - name: Configure
     shell: bash
     run: |
       echo "::group::CMake configuration"
-      mkdir build
       export PATH=$PWD/toolchain/bin/:$PATH
-      cmake -GNinja -B./build -S./llvm_test_suite -DTEST_SUITE_SUBDIRS=SYCL -DCHECK_SYCL_ALL="${{ inputs.check_sycl_all }}" -DCMAKE_CXX_COMPILER="$PWD/toolchain/bin/clang++" -DTEST_SUITE_LIT="$PWD/lit/lit.py" ${{ inputs.cmake_args }}
+      # TODO: Rename check_sycl_all input
+      cmake -GNinja -B./build-e2e -S./llvm/sycl/test-e2e -DSYCL_TEST_E2E_TARGETS="${{ inputs.check_sycl_all }}" -DCMAKE_CXX_COMPILER="$PWD/toolchain/bin/clang++" -DLLVM_LIT="$PWD/llvm/llvm/utils/lit/lit.py" ${{ inputs.cmake_args }}
```

Comment:
Where do we get ./llvm/ directory from? I see that we checkout only llvm-test-suite repository, but not llvm.

Reply:
Here: https://github.com/intel/llvm/blob/06d7bffdf53580b984859d27be157d17bde646b4/.github/workflows/sycl_linux_build_and_test.yml#L223-L227

inside the "caller" of this action.

Reply:
Wow... we checkout the whole llvm repository just to get access to two files: `./llvm/devops/actions/cleanup` and `./llvm/devops/actions/llvm_test_suite`. 

From [this commit message](https://github.com/intel/llvm/commit/ecb568edf79fbfeff4f2f80fc0a0d8fcc67c2bfa):

> For yet unknown reason, enabling cache doesn't work for intel/llvm repo in LLVM Test Suite jobs, but since LLVM Test Suite action will be packaged with containers in future, this is not a big deal.

It seems the plan was to remove this step from the action in the future.